docs/pages/aggregator.mdx

```diff
@@ -0,0 +1,173 @@
+# ERC7786 Aggregator
+
+## Why does it matter?
+
+When someone sends a cross-chain message using a given protocol, they're relying on its security, robustness and speed. This is ok for most retail users and use cases, but for some use cases, it might be necessary to add extra layers of security while handling with cross-chain messages.
+
+Since ERC7786 standardizes the way messages are sent and received on source and destination chains, although sent through different protocols, the messages must be the same on the destination chain, allowing one to send a message to various gateways and check if they are all were delivered properly.
```

Comment:
It's not clear what you meant to express here, can you elaborate and try different phrasing?